templates/vertical-grid/script/facets.hbs

```diff
@@ -8,5 +8,6 @@ ANSWERS.addComponent('Facets', {
       facetsDecorator.onMount(this);
     }
   },
+  transformFacets: {{{ generateTransformFacetsCall componentSettings.Facets }}},
```

Comment:
It feels odd that we have JavaScript returning a string so that our HBS can make JavaScript with that string. I know this is a limitation of our build and way out of the scope of your item. Just a note.

Reply:
It's definatly a little odd, but we can get around it by going with the approach in #669